be/src/exprs/vectorized/array_functions.h

```diff
@@ -40,6 +40,27 @@ class ArrayFunctions {
 
 #undef DEFINE_ARRAY_DISTINCT_FN
 
+#define DEFINE_ARRAY_SORT_FN(NAME, PT)                                                     \
```

Comment:
@mofeiatwork Hi, I think this macro could be simplified with your `type_dispatch_basic` in https://github.com/StarRocks/starrocks/pull/2995/files. right？

Reply:
may be difficult to implement by type_distpatch_basic;

Reply:
APPLY_FOR_TYPE_INTEGER should support variadic